src/app/users/users.component.html

```diff
@@ -1 +1,20 @@
-<p>users works!</p>
+<br />
+<div *ngFor="let u of users" class="card" style="width: 18rem">
```

Comment:
Don't use inline styles. move the code to the scss file of the component and apply a class

Reply:
It is a bootstrap template, If I do as you say, will it work the same? really dont know. thks

Reply:
fixed thanks